dkam/pkg/script/uos/cpio_build/etc/caddy/caddy_run.sh

```diff
@@ -56,30 +56,25 @@ if [ -z "$host_guid" ]; then
 fi
 
 
-# Update hosts if they were provided
-extra_hosts_needed=$(printf '%s\n' "$EXTRA_HOSTS" | sed "s|,|\n|g")
+# # Update hosts if they were provided
+# extra_hosts_needed=$(printf '%s\n' "$EXTRA_HOSTS" | sed "s|,|\n|g")
 
-printf '%s\n' "$extra_hosts_needed" >> /etc/hosts
-echo "Adding extra host mappings completed"
+# printf '%s\n' "$extra_hosts_needed" >> /etc/hosts
+# echo "Adding extra host mappings completed"
 
 export http_proxy="$http_proxy"
 export https_proxy="$https_proxy"
 export no_proxy="$no_proxy"
 
+export oci_release_svc="${oci_release_svc:-}"
```

Comment:
New environment variables (oci_release_svc, tink_stack_svc, etc.) are being exported with default empty values. If these are sometimes unset, this is a good practice. Make sure all subsequent scripts handle empty values correctly.